src/server/bootstrap/helmet.ts

```diff
@@ -0,0 +1,19 @@
+import helmet from 'helmet'
+import config from '../config'
+
+const cspOnlyReportViolations = config.get('cspOnlyReportViolations')
+const cspReportUri = config.get('cspReportUri')
+
+export default helmet({
+  contentSecurityPolicy: {
+    directives: {
+      defaultSrc: ["'self'"],
+      styleSrc: ["'self'", "'unsafe-inline'", 'https://fonts.googleapis.com/'],
```

Comment:
need to investigate further whether we use inline styles

Reply:
Mostly by React